custom_components/shopping_list_with_grocy/__init__.py

```diff
@@ -129,6 +129,31 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 
     return True
 
+    #
+    # To v4
+    #
+
+    if config_entry.version == 3:
+        v2_options = dict(config_entry.options)
+        v2_data = dict(config_entry.data)
```

Comment:
Could you use same syntax as above for variable initialization please?

Reply:
done